app/controllers/concerns/application_shared.rb

```diff
@@ -15,4 +15,12 @@ def set_timezone
     # Reset timezone for the next thread
     Time.zone = default_timezone
   end
-end
\ No newline at end of file
+
+  private
+
+  def method_missing(method_sym, *arguments, &block)
+    return super unless method_sym.to_s == "#{controller_name.singularize}_params"
+    params.require(controller_name.singularize.to_sym).
```

Comment:
I think so, I tested just by changing one test to be `post :create, valid_attrs` rather than `post :create, { :puppetclass => valid_attrs }`. I can't really recall but don't we test that unwrapped still works anywhere?